### PR TITLE
rename group collection

### PR DESCRIPTION
**Pull request by itkg-ngilain**

### CODE REVIEW

File: UserBundle/Document/Group.php [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 /**
  * @ODM\Document(
- *  collection="group_document",
+ *  collection="group",
```
Comment (nicolasThal) [inline UserBundle/Document/Group.php]: this should not be done, if you do it, you cannot run db.group.find() in the mongo shell

Comment (itkg-ngilain) [inline UserBundle/Document/Group.php]: Ok I understand now why this strange naming compared to the others documents.
I'm not a fan of group_document as it's the only collection to have such naming, so can I propose an alternative such as users_group or rights_group ? It avoids the basic naming conflict plus the unique and artificial suffix "document", but upgrades the meaning.

**Comment by itkg-ngilain** [comment]
@nicolasThal : update done

**Comment by nicolasThal** [comment]
:+1:, do not forget the changelog entry

**Pull request merged**